src/sql/services/bootstrap/bootstrapService.ts

```diff
@@ -94,6 +94,7 @@ export interface IBootstrapService {
 	commandService: ICommandService;
 	dashboardWebviewService: IDashboardWebviewService;
 	jobManagementService: IJobManagementService;
+	agentJobCacheService: IAgentJobCacheService;
```

Comment:
I think the cache should be integrated directly into the JobManagementService, perhaps as a composed object, for example.